tests/TestCadQuery.py

```diff
@@ -2059,3 +2059,17 @@ def testFindSolid(self):
         s = r.findSolid()
         self.assertEqual(len(s.Solids()),2)
         self.assertTrue(isinstance(s,Compound))
+
+    def testSlot(self):
+
+        decimal_places = 9
+
+        #Ensure it produces a solid with the correct volume
+        result = Workplane("XY").slot(4,1,0).extrude(1)
```

Comment:
Might be worth adding [cutBlind](https://cadquery.readthedocs.io/en/latest/classreference.html?highlight=cutblind#cadquery.Workplane.cutBlind) and/or [cutThruAll](https://cadquery.readthedocs.io/en/latest/classreference.html?highlight=cutthruall#cadquery.Workplane.cutThruAll) tests since I would expect that's normally how slots would be used, and on rare occasions extrude suceeds when a cut doesn't.

Reply:
That is a good point, and cutBlind() actually does reveal a bug.
```
result = cq.Workplane("XY").box(5,5,1).faces(">Z").workplane() \
             .slot(4,1,0).cutBlind(-0.5)
```
The above code does not produce the expected result. For some reason, the slot is offset by 0.5 above the surface of the box. Further testing shows that it is always offset above the top of the box by half the box height.

Reply:
Ah, it is caused by the cast of pnt to world coordinates.